libs/utils/include/celix/Filter.h

```diff
@@ -118,6 +118,32 @@ namespace celix {
             return celix_filter_findAttribute(cFilter.get(), attributeKey.c_str()) != nullptr;
         }
 
+        /**
+         * @brief Check whether the filter indicates the mandatory presence of an attribute with a specific value for the provided attribute key.
+         *
+         * Example:
+         *   using this method for attribute key "key1" on filter "(key1=value1)" yields true.
+         *   using this method for attribute key "key1" on filter "(!(key1=value1))" yields false.
+         *   using this method for attribute key "key1" on filter "(key1>=value1)" yields false.
+         *   using this method for attribute key "key1" on filter "(|(key1=value1)(key2=value2))" yields false.
+         */
+        [[nodiscard]] bool hasMandatoryEqualsValueAttribute(const std::string& attributeKey) const {
```

Comment:
Can the `const std::string&` be changed to `std::string_view`. This way the method accepts `std::string`, `const char*`, `std::string_view` and even constexpr `std::string_view`.

Something like:
```
[[nodiscard]] bool hasMandatoryEqualsValueAttribute(std::string_view attributeKey) const {
    return celix_filter_hasMandatoryEqualsValueAttribute(cFilter.get(), attributeKey.data());
}
``` 

same for `hasMandatoryNegatedPresenceAttribute`

Reply:
changed to std::string_view